app/models/manageiq/api/common/application_record.rb

```diff
@@ -0,0 +1,9 @@
+module ManageIQ
+  module API
+    module Common
+      class ApplicationRecord < ActiveRecord::Base
```

Comment:
Do we need a Common::ApplicationRecord - feels weird...I would just expect mixins.  Actually this is going to get weird in general if models are in 2 places, because you can't have 2 ApplicationRecords.  cc @syncrou